src/index.js

```diff
@@ -9,6 +9,8 @@ import passport from 'passport';
 import Routes from './routes/v1';
 import './helpers/passport/google';
 import './helpers/passport/github';
+// import userRoutes from './routes/user.route';
```

Comment:
Please remove this comments

Reply:
Thank you for this review. This has been resolved.